charts/perses/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: perses
 description: Perses helm chart
 icon: https://avatars.githubusercontent.com/u/77209215?s=200&v=4
 type: application
-version: 0.3.0
-appVersion: "0.42.1"
+version: 1.0.0
```

Comment:
My reasoning here for 1.0.0, the changes I made are technically not backward compatible with older versions from a semver.org perspective it requires a bump of a major version. If you want a different version bump here to indicated some fragility here, I say yes to any suggestions you want make here :)

Reply:
from the semver point of view, v0 is showing that the project is still under development. Under a v0, it doesn't need to be backward compatible, so it's ok to just bump the version to `0.4.0`